spec/fake_braintree/credit_card_spec.rb

```diff
@@ -77,6 +77,13 @@ def token_for(month, year)
     end
   end
 
+  it "sets the creation time" do
+    credit_card = Braintree::CreditCard.create(build_credit_card_hash).credit_card
+
+    creation_time = Time.parse(credit_card.created_at)
```

Comment:
Do not use `Time.parse` without zone. Use one of `Time.zone.parse`, `Time.parse.current`, `Time.parse.in_time_zone`, `Time.parse.utc`, `Time.parse.getlocal`, `Time.parse.iso8601`, `Time.parse.jisx0301`, `Time.parse.rfc3339`, `Time.parse.to_i`, `Time.parse.to_f` instead.